--- system-config-printer-1.3.9/udev/udev-configure-printer.c~	2011-10-26 14:14:20.000000000 +0200
+++ system-config-printer-1.3.9/udev/udev-configure-printer.c	2012-06-02 17:58:49.649600371 +0200
@@ -500,7 +500,7 @@
       *dest = '\0';
     }
 
-  sys = udev_get_sys_path (udev);
+  sys = "/sys";
   syslen = strlen (sys);
   devpathlen = strlen (devpath);
   syspath = malloc (syslen + devpathlen + 1);
